crates/polars-core/src/series/implementations/datetime.rs

```diff
@@ -356,20 +356,32 @@ impl SeriesTrait for SeriesWrap<DatetimeChunked> {
             .max_as_series()
             .into_datetime(self.0.time_unit(), self.0.time_zone().clone()))
     }
+
     fn min_as_series(&self) -> PolarsResult<Series> {
         Ok(self
             .0
             .min_as_series()
             .into_datetime(self.0.time_unit(), self.0.time_zone().clone()))
     }
+
+    fn median_as_series(&self) -> PolarsResult<Series> {
+        Series::new(self.name(), &[self.median().map(|v| v as i64)]).cast(self.dtype())
+    }
+
+    fn var_as_series(&self, _ddof: u8) -> PolarsResult<Series> {
+        Int32Chunked::full_null(self.name(), 1).cast(self.dtype())
+    }
+
+    fn std_as_series(&self, _ddof: u8) -> PolarsResult<Series> {
+        Int32Chunked::full_null(self.name(), 1).cast(self.dtype())
+    }
+
     fn quantile_as_series(
         &self,
         _quantile: f64,
         _interpol: QuantileInterpolOptions,
     ) -> PolarsResult<Series> {
-        Ok(Int32Chunked::full_null(self.name(), 1)
-            .cast(self.dtype())
-            .unwrap())
+        Int32Chunked::full_null(self.name(), 1).cast(self.dtype())
```

Comment:
Not part of my PR, but was in an adjacent function and small enough not to warrant another PR. No need to unwrap/rewrap.